src/main/java/fr/esgi/al5_2/Tayarim/dto/proprietaire/AdministrateurUpdateDTO.java

```diff
@@ -0,0 +1,19 @@
+package fr.esgi.al5_2.Tayarim.dto.proprietaire;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+
+@Data // Generates getters, setters, toString, equals, and hashCode methods
+@AllArgsConstructor // Generates an all-args constructor
+@Builder
+public class AdministrateurUpdateDTO {
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.naming.AbbreviationAsWordInNameCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Abbreviation in name 'AdministrateurUpdateDTO' must contain no more than '1' consecutive capital letters.